sushi_library/library/models.py

```diff
@@ -56,3 +57,75 @@ class Meta:
                              on_delete=models.CASCADE)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+
+class event(models.Model):
+    """Model representing a event ."""
+    title = models.CharField(max_length=200)
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, help_text='Unique ID for this particular book across whole library')
+    eventtype = models.CharField(
+        max_length=10,
+        choices=EVENT_STATUS,
+        blank=True,
+        default='r',
+        help_text='Book availability',
+    )
+    
+    
+EVENT_STATUS = (
+        ('r', 'requested'),
+        ('ar', 'approve the requesting'),
+        ('rr', 'requesting rejected'),
+        ('b', 'borrowed by someone'),
+        ('s', 'stored to lab')
+    )
+   
+    
+   
+
+   
+
+    def __str__(self):
+        """String for representing the Model object."""
+        return self.title
+
+
+
+class event_type (models.Model):
+    """Model representing an event."""
+    
+    bookname = models.CharField(max_length=100)
+    imprint = models.CharField(max_length=200)
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, help_text='Unique ID for this particular book across whole library')
+    requested_date = models.DateField(null=True, blank=True)
+    
+    eventtype = models.CharField(
+        max_length=10,
+        choices=EVENT_STATUS,
+        blank=True,
+        default='r',
+        help_text='Book availability',
+    )
+    
+    
+EVENT_STATUS = (
+        ('r', 'requested'),
+        ('ar', 'approve the requesting'),
+        ('rr', 'requesting rejected'),
+        ('b', 'borrowed by someone'),
+        ('s', 'stored to lab')
+    )
+
+
+
+    class Meta:
+        ordering = ['requested_date']
```

Comment:
Put Meta class on the next line after `class event_type (models.Model):`.
`ordering` is not needed.
`db_table` is needed.